library/components/TUrl.vue

```diff
@@ -47,11 +47,13 @@ export default {
   computed: {
     fullUrl() {
       let allUrlParams = {};
+      const allFilters = {};
+      this.filters.forEach((f) => (allFilters[f.name] = f.value));
       const {
         "context[org]": org,
         "context[group]": group,
         ...restOfFilters
-      } = this.getFiltersState;
+      } = allFilters;
```

Comment:
In order to avoid future issues, we should not access `this.filters` directly but use the page's store getter instead: `getFiltersState`.
Then we can use a `reduce` call to create this object.